src/module.ts

```diff
@@ -37,9 +37,9 @@ export default defineNuxtModule<NuxtI18nOptions>({
     if (isNuxt2(nuxt)) {
       throw new Error(
         formatMessage(
-          `We will release >=7.3 <8, See about GitHub Discussions https://github.com/nuxt-community/i18n-module/discussions/1287#discussioncomment-3042457: ${getNuxtVersion(
+          `v8 does not support Nuxt 2 and you're using ${getNuxtVersion(
```

Comment:
Is it useful to print specific Nuxt version? Why not just say that Nuxt 2 is not supported since we know this is running under Nuxt 2?

Reply:
True, I thought about removing it, but decided to keep it because @kazupon put it there and I didn't want to interfere with that. I can surely remove it.